templates/react-js/src/index.js

```diff
@@ -0,0 +1,13 @@
+/// <reference types="jquery" />
+/// <reference types="@geps/geofs-types" />
+
+import { createRoot } from "react-dom/client";
+import React from "react";
+import { App } from "./App";
```

Comment:
change something in the code linting because lib imports before local files imports are more important than alphabetical imports

Reply:
this could be an eslint-plugin-import thing instead of the built-in sort-imports rule

Reply:
that does extend outside of the scope of this PR though, so I'm gonna turn off sort-imports for now, and open a pr to use eslint-plugin-import in all the templates after this merges.